src/tactic/monotonicity/interactive.lean

```diff
@@ -405,7 +405,7 @@ open monad
 
 /-- tactic-facing function, similar to `interactive.tactic.generalize` with the
 exception that meta variables -/
-meta def generalize' (h : name) (v : expr) (x : name) : tactic (expr × expr) :=
+meta def monotonicity.generalize' (h : name) (v : expr) (x : name) : tactic (expr × expr) :=
```

Comment:
```suggestion
private meta def monotonicity.generalize' (h : name) (v : expr) (x : name) : tactic (expr × expr) :=
```
I'm not exactly sure what the difference to the standard `generalize` is, the docstring here is only partially illuminating.